mne/io/ctf/info.py

```diff
@@ -73,6 +73,11 @@ def _pick_isotrak_and_hpi_coils(res4, coils, t):
 
 def _convert_time(date_str, time_str):
     """Convert date and time strings to float time."""
+    if date_str == time_str == '':
+        date_str = '01/01/2000'
```

Comment:
```suggestion
        date_str = '01/01/1970'
```
This is the epoch and should translate to a `meas_date` of `(0., 0.)`. I'll push a commit to test this code path, too